.github/workflows/go.yml

```diff
@@ -109,11 +109,13 @@ jobs:
     strategy:
```

Comment:
I think the nightly jobs should go into our cron run (go-scheduled.yml) not in the tests that run on all PRs. Reason: we don't want PR merges blocked because of some incompatibility introduced by timescaledb. But, we do want to find out if such problems have been created